lightning/restore/restore.go

```diff
@@ -1158,6 +1188,10 @@ func (t *TableRestore) importEngine(
 	if err != nil {
 		return errors.Trace(err)
 	}
+	err = closedEngine.Cleanup(ctx)
```

Comment:
I think it's simpler to move `saveStatusCheckpoint`  into importKV before engine.Cleanup